tests/io/cif.cpp

```diff
@@ -143,4 +167,45 @@ TEST_F(ImportCIFTest, CuBTC)
     cifHandler.setRemoveAtomics(true);
     EXPECT_EQ(cifHandler.molecularSpecies().size(), 0);
 }
+
+TEST_F(ImportCIFTest, MoleculeOrdering)
+{
+    CIFHandler cifHandler;
+    const auto cifFiles = {"cif/molecule-test-simple-ordered", "cif/molecule-test-simple-unordered",
+                           "cif/molecule-test-simple-unordered-rotated"};
+    for (auto cifFile : cifFiles)
+    {
+        // Load the CIF file
+        ASSERT_TRUE(cifHandler.read(fmt::format("{}.cif", cifFile)));
```

Comment:
```suggestion
    const auto cifFiles = {"cif/molecule-test-simple-ordered.cif", "cif/molecule-test-simple-unordered.cif",
                           "cif/molecule-test-simple-unordered-rotated.cif"};
    for (auto cifFile : cifFiles)
    {
        // Load the CIF file
        ASSERT_TRUE(cifHandler.read(cifFile));
```

I thought that this might be a bit clearer, though there might be a deeper meaning that I missed behind separating the extension.

Reply:
Another good catch.  This was left-over from when the testing compared against a reference XYZ file on disk, but since now we just use the cleaned unit cell coordinates as the reference the extensions don't need to be swapped.